BookPlayer/Jellyfin/JellyfinCoordinator.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  JellyfinCoordinator.swift
+//  BookPlayer
+//
+//  Created by Lysann Tranvouez on 2024-10-26.
+//  Copyright © 2024 Tortuga Power. All rights reserved.
+//
+
+import Combine
+import Foundation
+import JellyfinAPI
+import UIKit
+
+class JellyfinCoordinator: Coordinator {
+  let flow: BPCoordinatorPresentationFlow
+  private let singleFileDownloadService: SingleFileDownloadService
+  private var disposeBag = Set<AnyCancellable>()
+
+  private var apiClient: JellyfinClient?
+  private var userID: String?
+  private var libraryName: String?
```

Comment:
I'll have another look at this.

On that note, I planned to keep it simple and have just one saved connection for simplicity. If other services are ever integrated, it could make more sense to allow several jellyfin connections too, since it would just be one more connection in a list that needs to be managed anyway. But I think for this PR I would rather keep it simple. But as I said, I'll have another look at those optionals.

Reply:
keeping the scope to just one connection for this pr sounds good, just wondering about the logout to that connection, is that meant to be handled in a separate PR?

Reply:
I plan to add them to this PR.